include/benchmark/reporter.h

```diff
@@ -85,8 +85,9 @@ class BenchmarkReporter {
     double max_heapbytes_used;
 
     // Keep track of arguments to compute asymptotic complexity
-    BigO   complexity;
-    int complexity_n;
+    BigO complexity;
+    BigOFunc* complexity_lambda;
+    size_t complexity_n;
```

Comment:
this should be int. google style (https://google.github.io/styleguide/cppguide.html) prefers int.